### DISCUSSION TITLE
How do I pipe the results from PsFzf into another command?

### DISCUSSION THREAD

**Discussion by gmkado** [kelleyma49/PSFzf#191] [Q&A] [answered]
This is probably just me not understanding how to use powershell properly.  For example I am trying to use the output of `Invoke-PsFzfGitBranches` to feed into `git checkout`.  I've tried
```
Invoke-PsFzfGitBranches | git checkout
```
and 
```
Invoke-PsFzfGitBranches | % { git checkout $_ }
```
and neither seem to work.  It just prints the name of the branch after the command:
```
Invoke-PsFzfGitBranches | % { git checkout $_ }feature/my_selected_branch
```

**Comment by kelleyma49** [comment, marked as answer]
It was a bad design on my part. Here's the remedy: https://github.com/kelleyma49/PSFzf/pull/193

**Reply by kelleyma49** [reply]
New alpha version is being published with my change, and will be available here: https://www.powershellgallery.com/packages/PSFzf/2.5.13-alpha

**Comment by gmkado** [comment]
@kelleyma49 you are awesome, thanks for the update! Now I can do something like ` git checkout $(Invoke-PsFzfGitBranches)`
